test/ownership/HasNoTokens.test.js

```diff
@@ -3,17 +3,17 @@ const { expectThrow } = require('../helpers/expectThrow');
 const HasNoTokens = artifacts.require('HasNoTokens');
 const ERC223TokenMock = artifacts.require('ERC223TokenMock');
 
-contract('HasNoTokens', function (accounts) {
+contract('HasNoTokens', function ([_, owner, initialAccount, anyone]) {
   let hasNoTokens = null;
   let token = null;
 
   beforeEach(async () => {
     // Create contract and token
-    hasNoTokens = await HasNoTokens.new();
-    token = await ERC223TokenMock.new(accounts[0], 100);
+    hasNoTokens = await HasNoTokens.new({ from: owner });
+    token = await ERC223TokenMock.new(initialAccount, 100);
```

Comment:
Hi @nventuro, visiting from `solidity-coverage`. 

Is it possible an assumption changed here? The first ERC223 constructor arg is not identical to the `from` address of the HasNoTokens creation as before. 

The [test below](https://github.com/OpenZeppelin/openzeppelin-solidity/pull/1137/files#diff-01929739950f027082f56ccf6991ba3eL21) (for the missing coverage) just throws an error - maybe it's a different one? 

We haven't published any changes over there for a bit and all your recent CI runs look good. . .

Reply:
Ha, I think you're right! Only `InitialAccount` will have balance, and the other test fails on the `transfer` call. Thank you so much!